tests/Magick.NET.Tests/Statistics/PerceptualHashTests/TheSumSquaredDistanceMethod.cs

```diff
@@ -1,15 +1,38 @@
 ﻿// Copyright Dirk Lemstra https://github.com/dlemstra/Magick.NET.
 // Licensed under the Apache License, Version 2.0.
 
+using System;
 using ImageMagick;
 using Xunit;
 
 namespace Magick.NET.Tests;
 
 public partial class PerceptualHashTests
 {
+    public class TestPerceptualHash : IPerceptualHash
```

Comment:
I will move this to a separate file after merging this PR. Not yet sure where I want to put this. And maybe we could use `NSubstitute` instead?

Reply:
I'm used to anonymous object that can implements an Interface in PHP or Kotlin 😜

I don't know `NSubstitute` ([this one](https://nsubstitute.github.io/)?), but I could use it instead of this Test class.